monai/apps/nnunet/nnunetv2_runner.py

```diff
@@ -522,38 +523,33 @@ def train_single_model(self, config: Any, fold: int, gpu_id: tuple | list | int
             kwargs.pop("export_validation_probabilities")
             logger.warning("please specify the `export_validation_probabilities` in the __init__ of `nnUNetV2Runner`.")
 
+        # from nnunetv2.run.run_training import run_training
+        cmd = self.train_single_model_command(config, fold, gpu_id, kwargs)
+        run_cmd(cmd, shell=True)
```

Comment:
There are a couple of environment variable settings in:
https://github.com/Project-MONAI/MONAI/blob/e26cab4cd1e340b839bbcfd1f2e4906599a3242f/monai/apps/nnunet/nnunetv2_runner.py#L190

Do we need to copy them to the environment variables in this subprocess run under `run_cmd`?

Reply:
It looks like it should be put in init, so that preprocessing, training, and inference can use.
What do you think?

Reply:
Hi @KumoLiu , to clarify my previous question, I mean maybe we need to:
```
ps_environ = os.environ.copy()
run_cmd(cmd.split(), env=ps_environ, ...)
```
I agree with your comment that they should stay where they were (init).

Reply:
Got it.
If env is None, it inherits the current process's environment. So it looks like we don't have to pass it?

Reply:
Thanks, that resolves my question!